.editorconfig

```diff
@@ -0,0 +1,17 @@
+# Drupal editor configuration normalization
```

Comment:
Providing this file in a template allows to have a "place" to put additional editor configurations.